src/process.rs

```diff
@@ -150,21 +150,34 @@ impl Runnable for Process {
                         thread::sleep(time::Duration::from_millis(10));
 
                         let mut p = lockable.lock().unwrap();
+                        // p is a MutexGuard<_Process>, so each access to fields of _Process calls
+                        // deref / deref_mut to get the underlying _Process. To avoid borrowing
+                        // conflicts, get a reference to the underlying _Process struct once.
+                        let p = &mut *p;
 
                         // Handle stdio
                         if let Some(ref mut stdout) = p.stdout_reader {
                             for line in stdout.lines() {
-                                info!("{:?}", line)
+                                p.stdout_sender
```

Comment:
After https://github.com/fubarnetes/fubarnetes/issues/1 is solved, this should ideally be some `PUB`/`SUB` or similar message queue.